### PR TITLE
Bug Fixes

### PR DESCRIPTION
**Pull request by SimoneSchmidt**
- Fixed the way the WizardStepper is integrated in StartTaskModal and EditScheduledEventsModal, so that the navigation buttons on top work as intended.
 - Changed capture agent drop-downs to searchable drop-downs.
 - Changed time drop-downs in EventDetailsSchedulingTab to searchable drop-downs.

### CODE REVIEW

File: modules/admin-ui-picard/app/src/components/events/partials/ModalTabsAndPages/EditScheduledEventsEditPage.js [hunk 1]
```diff
@@ -137,7 +112,8 @@ const EditScheduledEventsEditPage = ({ previousPage, nextPage, formik, inputDevi
                                                                             <span>{t('EVENTS.EVENTS.DETAILS.METADATA.TITLE')}</span>
                                                                         </td>
                                                                         <td className="editable ng-isolated-scope">
-                                                                            <Field name={`editedEvents.${key}.changedTitle`}
+                                                                            <Field tabIndex={(key * 14) + 1}
```
Comment (isabellakutger) [inline modules/admin-ui-picard/app/src/components/events/partials/ModalTabsAndPages/EditScheduledEventsEditPage.js:120]: leave here a comment, why you calculate the tab index this way

**Review by isabellakutger — CHANGES_REQUESTED** [review]
- Search in the language field in wizards and modals is not working
- Fix style of dropdowns, currently all dropdowns have different styles (both searchable ones and non-searchable ones)
-

**Review by isabellakutger — CHANGES_REQUESTED** [review]
- The dropdowns in the new event wizard for scheduling are still not searchable. The same goes for the edit scheduled events bulk modal.
- The style is better now, but in some places, there are still the old OC dropdowns and in some the plain HTML ones. Moreover, the searchable dropdowns should be a lot narrower to go with the style of the non-searchable ones

**Review by isabellakutger — APPROVED** [review]

**Comment by SimoneSchmidt** [comment]
> * The dropdowns in the new event wizard for scheduling are still not searchable. The same goes for the edit scheduled events bulk modal.
> 
>  * The style is better now, but in some places, there are still the old OC dropdowns and in some the plain HTML ones. Moreover, the searchable dropdowns should be a lot narrower to go with the style of the non-searchable ones

The remaining drop-downs will all be changed to the new drop-down component in the future.

**Pull request merged**